features/borrowers_personal_info.feature

```diff
@@ -0,0 +1,435 @@
+           "forename": "Paul",
+           "middle_name": "",
+           "surname": "Smythe",
+           "gender": "M",
+           "address": "1 The High Street Highley PL6 7TG",
+           "dob": "01/10/1976",
+           "phone_number": "07507154072, 07528670998"
+       }
+    ]
+    }
+    """
+
+    Then a status code of "400" is returned
+    And a message for failure is given "Failed validating 'pattern' in schema['properties']['borrowers']['items']['properties']['phone_number']:"
+
+Scenario: Create a deed without a mobile phone number
+    Given I add the following deed:
+
+    """
+    {
+    "title_number": "DT567568",
+    "borrowers": [
+      {
+           "forename": "Paul",
+           "middle_name": "",
+           "surname": "Smythe",
+           "gender": "M",
+           "address": "1 The High Street Highley PL6 7TG",
+           "dob": "01/10/1976",
+           "phone_number": ""
+       }
+    ]
+    }
+    """
+    Then a status code of "400" is returned
+    And a message for failure is given "Failed validating 'pattern' in schema['properties']['borrowers']['items']['properties']['phone_number']:"
+
+Scenario: Create a deed where both borrowers have a unique mobile phone number
+    Given I add the following deed:
+
+    """
+    {
+    "title_number": "DT567568",
+    "borrowers": [
+       {
+           "forename": "Katie",
+           "middle_name": "Elizabeth",
+           "surname": "Smythe",
+           "gender": "M",
+           "address": "1 The High Street Highley CR0 9YH",
+           "dob": "11/01/2000",
+           "phone_number": "07507154077"
+       },
+       {
+           "forename": "Paul",
+           "middle_name": "",
+           "surname": "Smythe",
+           "gender": "M",
+           "address": "1 The High Street Highley PL6 7TG",
+           "dob": "01/10/1976",
+           "phone_number": "07502154062"
+       }
+    ]
+    }
+    """
+
+    Then a status code of "201" is returned
+    And a url link to retrieve the deed is returned
+
+Scenario: Create a deed where both borrowers have the same mobile phone number
+    Given I add the following deed:
+    """
+    {
+    "title_number": "DT567568",
+    "borrowers": [
+       {
+           "forename": "Katie",
+           "middle_name": "Elizabeth",
+           "surname": "Smythe",
+           "gender": "F",
+           "address": "1 The High Street Highley CR0 9YH",
+           "dob": "11/01/2000",
+           "phone_number": "07507154077"
+       },
+       {
+           "forename": "Paul",
+           "middle_name": "",
+           "surname": "Smythe",
+           "gender": "blah",
+           "address": "1 The High Street Highley PL6 7TG",
+           "dob": "01/10/1976",
+           "phone_number": "07507154077"
+       }
+    ]
+    }
+    """
+
+    Then a status code of "400" is returned
+    And a message for failure is given "Failed validating 'pattern' in schema['properties']['borrowers']['items']['properties']['gender']:"
```

Comment:
Should the error message be complaining about gender when the issue is with the fact that the phone numbers aren't unique? Might want to double check this is correct.